metricbeat/docker-compose.yml

```diff
@@ -46,13 +47,26 @@ services:
     volumes:
       - ${PWD}/..:/go/src/github.com/elastic/beats/
       - /var/run/docker.sock:/var/run/docker.sock
+      - datavolume:/etc/ceph
```

Comment:
I'll study the rest api but I think that we can remove all this and don't use command any more.